lumi/connect.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/bash -ex
 #usage $0 <OS> <connection-string> e.g. connect.sh el8 username@host.domain
 OS=$1
-CONNECTION=$2
+TARGET=$2
 
-echo "Launching LUMI node... for ${OS} using ${CONNECTION}"
+# Values from get_slot.sh
+USER=cmsbuild
+SESSION=$OS
+SLURM_ACCOUNT=project_462000245
+SCRATCH=/scratch/$SLURM_ACCOUNT/$USER/$SESSION
+
+KTAB=${HOME}/keytabs/$(echo $TARGET | sed 's|@.*||').keytab
+if [ ! -f $KTAB ] ; then KTAB=${HOME}/keytabs/cmsbld.keytab ; fi
+export KRB5CCNAME=FILE:/tmp/krb5cc_$USER
```

Comment:
@aandvalenzuela , change this to 
```
export KRB5CCNAME=FILE:/tmp/krb5cc_${USER}_${NODE_NAME}
```
this will make sure that locally generated krb5 token is unique for every node